rustls/tests/common/mod.rs

```diff
@@ -523,6 +541,31 @@ pub fn make_server_config_with_client_verifier(
         .unwrap()
 }
 
+pub fn make_server_config_with_rpk_support_and_version(
+    kt: KeyType,
+    version: &[&'static rustls::SupportedProtocolVersion],
+) -> ServerConfig {
+    let mut client_verifier = MockClientVerifier::new(|| Ok(ClientCertVerified::assertion()), kt);
+    client_verifier.expect_raw_public_keys = true;
+    let client_verifier = Arc::new(client_verifier);
```

Comment:
nit: this last binding doesn't add much value. Suggest using `with_client_cert_verifier(client_verifier.into())` or moving the `Arc::new()` inline with `with_client_cert_verifier()`.